htdocs/admin/pdf.php

```diff
@@ -277,6 +277,12 @@
 
     print '<tr class="oddeven"><td>'.$langs->trans("HideDetailsOnPDF").'</td><td>';
     print $form->selectyesno('MAIN_GENERATE_DOCUMENTS_HIDE_DETAILS', (! empty($conf->global->MAIN_GENERATE_DOCUMENTS_HIDE_DETAILS))?$conf->global->MAIN_GENERATE_DOCUMENTS_HIDE_DETAILS:0, 1);
+    print '</td></tr>';
+
+	//Invert send and recipient
+
+	print '<tr class="oddeven"><td>'.$langs->trans("HideDetailsOnPDF").'</td><td>';
```

Comment:
The label "HideDetailsOnPDF" seems wrong.

Reply:
Corrected!